internal/resource/statefulset.go

```diff
@@ -598,6 +614,9 @@ func (builder *StatefulSetBuilder) podTemplateSpec(annotations, labels map[strin
 							SubPath:   "default_user.conf",
 						},
 					},
+					ImagePullPolicy:          corev1.PullIfNotPresent,
```

Comment:
We should not have an opinion here. This may surprise any user relying on `rabbitmq:latest` image.